scripts/validate.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# do basic validations about AMI generated
+
+#######################################
+# Validates file(or blob) don't exists
+# Arguments:
+#   an file name or blob
+# Returns:
+#   None
```

Comment:
```
# Validates that a file or blob doesn't exist
#
# Arguments:
#   a file name or blob
# Returns:
#   1 if a file exists, after printing an error
```

Reply:
humm, it's not actually return but will exit the script with 1(error) :D , but fixed 👀

Reply:
I meant that the whole validate.sh script exits with 1 😄